sigulsign_unsigned.py

```diff
@@ -240,14 +243,14 @@ def __init__(self, key, password=None, config_file=None, arch=None,
         self.key = key
 
         if password is None:
-            try:
-                krb_realm = "RPMFUSION.ORG"
-                if use_staging:
-                    krb_realm = "STAGING.RPMFUSION.ORG"
-
-                fas_username = cccolutils.get_user_for_realm(krb_realm)
-            except:
-                fas_username = getpass.getuser()
+#            try:
+#                krb_realm = "RPMFUSION.ORG"
+#                if use_staging:
+#                    krb_realm = "STAGING.RPMFUSION.ORG"
+#
+#                fas_username = cccolutils.get_user_for_realm(krb_realm)
+#            except:
+            fas_username = getpass.getuser()
 
```

Comment:
You can keep this part if (once) we will use kerberos. It shouldn't cause hard issue...